woocommerce/woocommerce-gutenberg-products-block
# Add new settings to Reviews by Product block

Part of #365 

This PR adds and updates the new block attributes specified in the [last designs](https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/588#issuecomment-512599607).

### Screenshots

![image](https://user-images.githubusercontent.com/3616980/62365001-3d64a680-b523-11e9-967d-b5032835abc8.png)

### How to test the changes in this Pull Request:

1. Create a post with a _Reviews by Product_ bock.
2. Try interacting with all settings and verify they work as expected.
3. In _Settings_ > _Discussion_ disable _Show Avatars_ and verify a notice appears when the image type is set to _Reviewer photo_ and the image is enabled.
4. In _WooCommerce_ > _Settings_ > _Products_ uncheck _Enable star rating on reviews_ and verify a notice appears when _Product rating_ is enabled.
